takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -4,69 +4,71 @@
 {% load datastructuretags %}
 
 {% block dropdown_id %}nav-about{% endblock %}
-
+<div class="text-right pr-3"></div>
 {% block dropdown_title %}About Takwimu{% endblock %}
 {% block dropdown_desc %}Takwimu provides quality insights and data-driven analysis to empower African development champions.{% endblock %}
 
 {% block dropdown_content %}
 <div class="row pl-4 pb-5" style="width: 100%;">
   <div class="col-sm-4">
-    <ul style="color:#7ab1f2; font-weight: bold;">
+    <ul style="color:#7ab1f2; font-weight: bold; line-height:2;">
```

Comment:
Remove the style attribute entirely by:
1. Use class `font-weight-bold` instead.
2. `line-height` and `color` is to be applied to **all** drop downs.